apps/node-host/src/bootstrap.js

```diff
@@ -6,24 +6,26 @@
 import express from 'express';
 import * as path from 'path';
 
-import node_local_remote from 'node_local_remote/test';
-
 const remoteMsg = import('node_remote/test').then((m) => {
   console.log('\x1b[32m%s\x1b[0m', m.default || m);
   return m.default || m;
 });
-console.log('\x1b[32m%s\x1b[0m', node_local_remote);
 
 const app = express();
 
 app.use('/assets', express.static(path.join(__dirname, 'assets')));
 
 app.get('/api', async (req, res) => {
+  const localRemoteMsg = import('node_local_remote/test').then((m) => {
```

Comment:
Why did you change a lot of node content in your mr

Reply:
in ci test , the node_local_remote may not compiled while the hots has requested , so the ci will be failed